Cargo.toml

```diff
@@ -35,6 +35,7 @@ rocket = { default-features = false, optional = true, version = "0.5.0-rc.1" }
 serde = { default-features = false, optional = true, version = "1.0" }
 serde_json = { default-features = false, optional = true, version = "1.0" }
 tokio-postgres = { default-features = false, optional = true, version = "0.7" }
+rkyv = {version = "0.7", optional = true}
```

Comment:
nit: can we do a couple of things here:
1. this list is alphabetical - can we put it before `rocket`? The parameters are also alphabeticalized.
2. Can we set `default-features = false` and include any required features explicitly?